examples/trace/main.go

```diff
@@ -37,4 +38,10 @@ func bar(ctx context.Context) {
 	defer span.End()
 
 	// Do bar...
+
+	// Set Datadog APM Trace Metadata
+	span.AddAttributes(
+		trace.StringAttribute(ddtraceext.ResourceName, "my-app-resource"),
+		// trace.StringAttribute(ddtraceext.SpanType, ddtraceext.SpanTypeWeb),
```

Comment:
Let's not leave any commented out code in examples.
```suggestion
		trace.StringAttribute(ddtraceext.SpanType, ddtraceext.SpanTypeWeb),
```